### ISSUE TITLE
Deprecation warning: moment construction falls back to js Date

### ISSUE THREAD

**Issue by danielspaniel** [offirgolan/ember-cp-validations#137]
I've just stared using cp-validations, and things are looking good. 
 Except for this issue for using date validator .. 
  I reckon that the latest moment js  has deprecated the use of moment(random date string)
  because when I use the date validator with before: 'now' or after: '2/2/2015' or whatever .. then I get these deprecation warnings from the base date validator class.
   
```
Deprecation warning: moment construction falls back to js Date. This is discouraged and will be removed in upcoming major release. Please refer to https://github.com/moment/moment/issues/1407 for more info.
Arguments: [object Object]
```

Is this already known issue?  Need me to try a PR? You feel like fixing?

**Comment by offirgolan** [comment]
A PR is always welcome! If not, I can always take a crack at it when I have some time :smile:

**Comment by danielspaniel** [comment]
fair enough mate. If I go for it will let you know, so we don't duplicate work.

**Comment by offirgolan** [comment]
:+1: Thanks @danielspaniel

**Comment by offirgolan** [comment]
Resolved with #140 (Thanks again!)
